src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.Component.targets

```diff
@@ -0,0 +1,336 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.Sdk.Razor.Component.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+
+<Project ToolsVersion="14.0">
+
+  <PropertyGroup>
+    <!-- Used for tracking inputs to component generation -->
+    <_RazorComponentInputHash></_RazorComponentInputHash>
+    <_RazorComponentInputCacheFile>$(IntermediateOutputPath)$(MSBuildProjectName).RazorComponent.input.cache</_RazorComponentInputCacheFile>
+
+    <_ResolveComponentsInputCache>$(IntermediateOutputPath)$(MSBuildProjectName).RazorComponent.definition.cache</_ResolveComponentsInputCache>
+
+    <_RazorComponentDeclarationManifest>$(IntermediateOutputPath)$(MSBuildProjectName).RazorComponents.declaration.json</_RazorComponentDeclarationManifest>
+    <_RazorComponentDefinitionManifest>$(IntermediateOutputPath)$(MSBuildProjectName).RazorComponents.definition.json</_RazorComponentDefinitionManifest>
+  </PropertyGroup>
```

Comment:
Should these files be shared with tag helpers? We want the build support to find it all.

Reply:
Good, that's what I was hoping to see.